extensions/amp-sidebar/1.0/amp-sidebar-1.0.md

```diff
@@ -58,103 +58,232 @@ limitations under the License.
+    - e.g `<nav toolbar="(max-width: 1024px)" target="toolbar-target">`
+- Toolbar behavior is only applied while the media-query is valid
 
-**layout**
-
-The only permissible value for the `layout` attribute in `amp-sidebar` is `nodisplay`.
+Example:
 
-**common attributes**
+```html
+<amp-sidebar id="sidebar1" layout="nodisplay" side="right">
+  <ul>
+    <li> Nav item 1</li>
+    <li> Nav item 2</li>
+    <li> Nav item 3</li>
+    <li> Nav item 4</li>
+    <li> Nav item 5</li>
+    <li> Nav item 6</li>
+  </ul>
 
-This element includes [common attributes](https://www.ampproject.org/docs/reference/common_attributes) extended to AMP components.
+  <nav toolbar="(max-width: 767px)">
+    <ul>
+      <li>
+        <input placeholder="Search..."/>
+      </li>
+    </ul>
+  </nav>
+</amp-sidebar>
+```
 
-## Styling
+Example (toolbar-only on `<nav>` element):
 
-The `amp-sidebar` component can be styled with standard CSS.
+```html
+<amp-sidebar id="sidebar1" layout="nodisplay" side="right">
+  <ul>
+    <li> Nav item 1</li>
+    <li> Nav item 2</li>
+    <li> Nav item 3</li>
+    <li> Nav item 4</li>
+    <li> Nav item 5</li>
+    <li> Nav item 6</li>
+  </ul>
 
-- The `width` of the `amp-sidebar` may be set to adjust the width of the sidebar between the pre-set min(45px) and max(80vw) values.
-- The height of the `amp-sidebar` may be set to adjust the height of the sidebar if required. If the height exceeds 100vw then the sidebar will have a vertical scrollbar. The preset height of the sidebar is 100vw and can be overridden in CSS to make it shorter.
-- The current state of the sidebar is exposed via the `open` attribute that is set on the `amp-sidebar` tag when the side bar is open on the page.
+  <nav toolbar="(min-width: 0px)" toolbar-only>
+    <ul>
+      <li>
+        <input placeholder="Search..."/>
+      </li>
+    </ul>
+  </nav>
+</amp-sidebar>
+```
 
-## Actions
-The `amp-sidebar` exposes the following actions you can use [AMP on-syntax to trigger](../../../spec/amp-actions-and-events.md):
+Example (toolbar-only on individual `<li>` elements):
 
-<table>
-  <tr>
-    <th>Action</th>
-    <th>Description</th>
-  </tr>
-  <tr>
-    <td>open (default)</td>
-    <td>Opens the sidebar</td>
-  </tr>
-  <tr>
-    <td>close</td>
-    <td>Closes the sidebar</td>
-  </tr>
-  <tr>
-    <td>toggle</td>
-    <td>Toggles the sidebar state</td>
-  </tr>
-</table>
+```html
+<amp-sidebar id="sidebar1" layout="nodisplay" side="right">
+  <ul>
+    <li> Nav item 1</li>
+    <li> Nav item 2</li>
+    <li> Nav item 3</li>
+    <li> Nav item 4</li>
+    <li> Nav item 5</li>
+    <li> Nav item 6</li>
+  </ul>
 
+  <nav toolbar="(min-width: 768px) and (max-width: 1024px)">
+    <ul>
+      <li>
+        Publisher Logo
+      </li>
+      <li>
+        <input placeholder="Search..." toolbar-only/>
+      </li>
+    </ul>
+  </nav>
+</amp-sidebar>
+```
 
-### Examples
+Example (target attribute):
```

Comment:
- Rewrite as "Example: Displaying the toolbar in the specified `target` element
- Add description of this example.